ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp

```diff
@@ -359,7 +359,7 @@ void TTable::Join( TTable & t1, TTable & t2, EJoinKind joinKind, bool hasMoreLef
             if (!HasBitSet(nullsPtr, 1))
             {
 
-                ui64 slotNum = hash % nSlots;
+                ui64 slotNum = (hash / NumberOfBuckets) % nSlots;
```

Comment:
Maybe `hash << BitsForNumberOfBuckets % nSlots`?

Reply:
On one hand, compiler optimizes divide-by-constant with proper combination of shift/muls/etc.
On other, if we make `BitsForNumberOfBuckets` runtime-configurable, compiler won't be able to optimize it.

In the end, I decided it is better to solve the problem differently, and choose `nSlots` to avoid collisions instead (this way, we need to spend more time once-per-resize instead of once-per-access, even if it is as trivial as bit-shift)